course_inventory/inventory.py

```diff
@@ -70,7 +70,9 @@ def make_request_using_api_utils(url: str, params: Union[Dict[str, Any], None] =
                 logger.info('Beginning next attempt')
 
     logger.error('The maximum number of request attempts was reached')
-    return response
+    logger.error(f'Data could not be gathered from the URL with the ending "{url}"')
+    logger.error('The program will exit')
```

Comment:
Will the whole program/cron exit, or just this single URL call exit? Please update the log message accordingly.

Reply:
The whole program will exit (thus I think the log message is accurate). I explained my reasoning in the PR description; @pushyamig seems to concur.

I suppose we might want other jobs (`MIVIDEO`, or `CANVAS_ZOOM_MEETINGS`) to continue running, but that would require a bunch more logic and modification of other functions. Keep in mind this is a failure case, and I have never seen it happen spontaneously.

Reply:
OK. Hope we won't encounter this failure.

Reply:
Possible the better thing to do if we just wanted to abort this job would be to raise a custom exception, catch it up in run_jobs. Then the other jobs could continue when this one fails. 

Jenkins would still consider this script a failure but it wouldn't block anything else. I'm fine if you want to consider this for the future.

Reply:
I think this is a good idea. I'll create a separate issue.